packages/gguf/src/gguf.spec.ts

```diff
@@ -12,8 +13,19 @@ const URL_V1 =
 	"https://huggingface.co/tmadge/testing/resolve/66c078028d1ff92d7a9264a1590bc61ba6437933/tinyllamas-stories-260k-f32.gguf";
 const URL_SHARDED_GROK =
 	"https://huggingface.co/Arki05/Grok-1-GGUF/resolve/ecafa8d8eca9b8cd75d11a0d08d3a6199dc5a068/grok-1-IQ3_XS-split-00001-of-00009.gguf";
+const URL_BIG_METADATA = "https://huggingface.co/ngxson/test_gguf_models/resolve/main/gguf_test_big_metadata.gguf";
```

Comment:
```suggestion
const URL_BIG_METADATA = "https://huggingface.co/ngxson/test_gguf_models/resolve/c72c0f045c36b7c01f348c5fbba424cd6a52f770/gguf_test_big_metadata.gguf";
```

to make the test case more robust, rather than `branch/file`, we can use `commit/file`